src/renderers/WebGLRenderer.js

```diff
@@ -2038,7 +2038,17 @@ function WebGLRenderer( parameters ) {
 		const levelScale = Math.pow( 2, - level );
 		const width = Math.floor( texture.image.width * levelScale );
 		const height = Math.floor( texture.image.height * levelScale );
-		const glFormat = utils.convert( texture.format );
+
+		let glFormat = utils.convert( texture.format );
+
+		if ( capabilities.isWebGL2 ) {
+
+			// Workaround for https://bugs.chromium.org/p/chromium/issues/detail?id=1120100
```

Comment:
Is it worth leaving a TOFIX in here saying that once https://crbug.com/882580 is fixed, this workaround should be removed?

Hate to suggest it, but should this workaround be limited to Chrome?

Reply:
> Is it worth leaving a TOFIX in here saying that once https://crbug.com/882580 is fixed, this workaround should be removed?

I'm subscribed to that bug. I'll remove the workaround as soon as it's fixed 👍

> Hate to suggest it, but should this workaround be limited to Chrome?

It doesn't seem to break other browsers so I think I'll keep the code simple for now.
I can add more checks if we notice any regressions.